hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4272,6 +4272,35 @@ func reconcileVirtLauncherNetworkPolicy(policy *networkingv1.NetworkPolicy, hclu
 			},
 		},
 	}
+	var nodeAddressesMap map[string]bool
```

Comment:
hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go:4285:3: assignment to nil map (SA5000)

Maybe you have to add `var nodeAddressesMap map[string]bool{}` with the `{}` at the end?